tsconfig.json

```diff
@@ -2,6 +2,6 @@
   "files": [],
   "references": [
     { "path": "./tsconfig.app.json" },
-    { "path": "./tsconfig.node.json" }
+    { "path": "./tsconfig.vite.json" }
```

Comment:
Im not sure this is an improvement to the vite to name one of these confids 'node' default, but I needed to understand the different purpose for these configs and I think it makes a little more sense. Maybe not though.